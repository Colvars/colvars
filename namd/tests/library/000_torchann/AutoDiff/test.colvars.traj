# step         nn_0                  r_nn_0                
           0    6.31129478791928e+01  6.31129478791928e+01 
           1    6.30795952608635e+01  6.31123487265859e+01 
           2    6.30231691808804e+01  6.31051168426741e+01 
           3    6.29423629424581e+01  6.30971510306278e+01 
           4    6.28368109214094e+01  6.30939566687055e+01 
           5    6.27071769949319e+01  6.30942455003346e+01 
           6    6.25551635807806e+01  6.30968143599985e+01 
           7    6.23834483072468e+01  6.30990150919339e+01 
           8    6.21955611390711e+01  6.30997820968386e+01 
           9    6.19957198408133e+01  6.30954650664551e+01 
          10    6.17886384901233e+01  6.30913040628081e+01 
          11    6.15793260812456e+01  6.30929329891023e+01 
          12    6.13728827706928e+01  6.30996441143896e+01 
          13    6.11742977675034e+01  6.31066980728339e+01 
          14    6.09882540781548e+01  6.31149728966408e+01 
          15    6.08189465919984e+01  6.31318656967496e+01 
          16    6.06699192830909e+01  6.31529303521710e+01 
          17    6.05439247654291e+01  6.31719915277253e+01 
          18    6.04428203768620e+01  6.31897506500413e+01 
          19    6.03675152477868e+01  6.32015820964369e+01 
          20    6.03179725903078e+01  6.32018194399742e+01 
