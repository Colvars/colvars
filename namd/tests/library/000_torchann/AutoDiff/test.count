# 1
# -1.80000000000000e+02  5.00000000000000e+00         72  0

 -1.77500000000000e+02                      0
 -1.72500000000000e+02                      0
 -1.67500000000000e+02                      0
 -1.62500000000000e+02                      0
 -1.57500000000000e+02                      0
 -1.52500000000000e+02                      0
 -1.47500000000000e+02                      0
 -1.42500000000000e+02                      0
 -1.37500000000000e+02                      0
 -1.32500000000000e+02                      0
 -1.27500000000000e+02                      0
 -1.22500000000000e+02                      0
 -1.17500000000000e+02                      0
 -1.12500000000000e+02                      0
 -1.07500000000000e+02                      0
 -1.02500000000000e+02                      0
 -9.75000000000000e+01                      0
 -9.25000000000000e+01                      0
 -8.75000000000000e+01                      0
 -8.25000000000000e+01                      0
 -7.75000000000000e+01                      0
 -7.25000000000000e+01                      0
 -6.75000000000000e+01                      0
 -6.25000000000000e+01                      0
 -5.75000000000000e+01                      0
 -5.25000000000000e+01                      0
 -4.75000000000000e+01                      0
 -4.25000000000000e+01                      0
 -3.75000000000000e+01                      0
 -3.25000000000000e+01                      0
 -2.75000000000000e+01                      0
 -2.25000000000000e+01                      0
 -1.75000000000000e+01                      0
 -1.25000000000000e+01                      0
 -7.50000000000000e+00                      0
 -2.50000000000000e+00                      0
  2.50000000000000e+00                      0
  7.50000000000000e+00                      0
  1.25000000000000e+01                      0
  1.75000000000000e+01                      0
  2.25000000000000e+01                      0
  2.75000000000000e+01                      0
  3.25000000000000e+01                      0
  3.75000000000000e+01                      0
  4.25000000000000e+01                      0
  4.75000000000000e+01                      0
  5.25000000000000e+01                      0
  5.75000000000000e+01                      0
  6.25000000000000e+01                     20
  6.75000000000000e+01                      0
  7.25000000000000e+01                      0
  7.75000000000000e+01                      0
  8.25000000000000e+01                      0
  8.75000000000000e+01                      0
  9.25000000000000e+01                      0
  9.75000000000000e+01                      0
  1.02500000000000e+02                      0
  1.07500000000000e+02                      0
  1.12500000000000e+02                      0
  1.17500000000000e+02                      0
  1.22500000000000e+02                      0
  1.27500000000000e+02                      0
  1.32500000000000e+02                      0
  1.37500000000000e+02                      0
  1.42500000000000e+02                      0
  1.47500000000000e+02                      0
  1.52500000000000e+02                      0
  1.57500000000000e+02                      0
  1.62500000000000e+02                      0
  1.67500000000000e+02                      0
  1.72500000000000e+02                      0
  1.77500000000000e+02                      0
