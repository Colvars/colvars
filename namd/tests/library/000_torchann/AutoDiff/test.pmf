# 1
# -1.82500000000000e+02  5.00000000000000e+00         73  0

 -1.80000000000000e+02   0.00000000000000e+00
 -1.75000000000000e+02   0.00000000000000e+00
 -1.70000000000000e+02   0.00000000000000e+00
 -1.65000000000000e+02   0.00000000000000e+00
 -1.60000000000000e+02   0.00000000000000e+00
 -1.55000000000000e+02   0.00000000000000e+00
 -1.50000000000000e+02   0.00000000000000e+00
 -1.45000000000000e+02   0.00000000000000e+00
 -1.40000000000000e+02   0.00000000000000e+00
 -1.35000000000000e+02   0.00000000000000e+00
 -1.30000000000000e+02   0.00000000000000e+00
 -1.25000000000000e+02   0.00000000000000e+00
 -1.20000000000000e+02   0.00000000000000e+00
 -1.15000000000000e+02   0.00000000000000e+00
 -1.10000000000000e+02   0.00000000000000e+00
 -1.05000000000000e+02   0.00000000000000e+00
 -1.00000000000000e+02   0.00000000000000e+00
 -9.50000000000000e+01   0.00000000000000e+00
 -9.00000000000000e+01   0.00000000000000e+00
 -8.50000000000000e+01   0.00000000000000e+00
 -8.00000000000000e+01   0.00000000000000e+00
 -7.50000000000000e+01   0.00000000000000e+00
 -7.00000000000000e+01   0.00000000000000e+00
 -6.50000000000000e+01   0.00000000000000e+00
 -6.00000000000000e+01   0.00000000000000e+00
 -5.50000000000000e+01   0.00000000000000e+00
 -5.00000000000000e+01   0.00000000000000e+00
 -4.50000000000000e+01   0.00000000000000e+00
 -4.00000000000000e+01   0.00000000000000e+00
 -3.50000000000000e+01   0.00000000000000e+00
 -3.00000000000000e+01   0.00000000000000e+00
 -2.50000000000000e+01   0.00000000000000e+00
 -2.00000000000000e+01   0.00000000000000e+00
 -1.50000000000000e+01   0.00000000000000e+00
 -1.00000000000000e+01   0.00000000000000e+00
 -5.00000000000000e+00   0.00000000000000e+00
  0.00000000000000e+00   0.00000000000000e+00
  5.00000000000000e+00   0.00000000000000e+00
  1.00000000000000e+01   0.00000000000000e+00
  1.50000000000000e+01   0.00000000000000e+00
  2.00000000000000e+01   0.00000000000000e+00
  2.50000000000000e+01   0.00000000000000e+00
  3.00000000000000e+01   0.00000000000000e+00
  3.50000000000000e+01   0.00000000000000e+00
  4.00000000000000e+01   0.00000000000000e+00
  4.50000000000000e+01   0.00000000000000e+00
  5.00000000000000e+01   0.00000000000000e+00
  5.50000000000000e+01   0.00000000000000e+00
  6.00000000000000e+01   0.00000000000000e+00
  6.50000000000000e+01   1.53701479101226e-01
  7.00000000000000e+01   1.53701479101226e-01
  7.50000000000000e+01   1.53701479101226e-01
  8.00000000000000e+01   1.53701479101226e-01
  8.50000000000000e+01   1.53701479101226e-01
  9.00000000000000e+01   1.53701479101226e-01
  9.50000000000000e+01   1.53701479101226e-01
  1.00000000000000e+02   1.53701479101226e-01
  1.05000000000000e+02   1.53701479101226e-01
  1.10000000000000e+02   1.53701479101226e-01
  1.15000000000000e+02   1.53701479101226e-01
  1.20000000000000e+02   1.53701479101226e-01
  1.25000000000000e+02   1.53701479101226e-01
  1.30000000000000e+02   1.53701479101226e-01
  1.35000000000000e+02   1.53701479101226e-01
  1.40000000000000e+02   1.53701479101226e-01
  1.45000000000000e+02   1.53701479101226e-01
  1.50000000000000e+02   1.53701479101226e-01
  1.55000000000000e+02   1.53701479101226e-01
  1.60000000000000e+02   1.53701479101226e-01
  1.65000000000000e+02   1.53701479101226e-01
  1.70000000000000e+02   1.53701479101226e-01
  1.75000000000000e+02   1.53701479101226e-01
  1.80000000000000e+02   1.53701479101226e-01
