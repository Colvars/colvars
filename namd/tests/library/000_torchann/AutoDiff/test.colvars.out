colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: Summary of compile-time features available in this build:
colvars:   - C++ language version: 201703
colvars:   - Multiple replicas: enabled (replica number 1 of 1)
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: available
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars: The final output state file will be "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "test.in":
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars:   prot_N (10 atoms)
colvars:   prot_CA (10 atoms)
colvars:   prot_C (10 atoms)
colvars:   prot_O (10 atoms)
colvars: # smp = "cvcs" [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # sourceTclFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "nn_0"
colvars:     Initializing a new "torchANN" component.
colvars:     Warning: cannot cite unknown feature "torchANN colvar component (derived from linearCombination)"
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 360 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group1"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 4 atoms requested: total mass = 54.028, total charge = -0.72.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group2"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:       Initializing atom group "group3".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group3"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group3".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group3" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:       Initializing atom group "group4".
colvars:       # name = "" [default]
colvars:       # centerToOrigin = off [default]
colvars:       # centerToReference = off [default]
colvars:       # rotateToReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "group4"
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group4".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group4" defined with 4 atoms requested: total mass = 54.028, total charge = -0.4.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:     # modelFile = "identity.pt"
colvars:     torch model loaded.
colvars:     # m_output_index = 0
colvars:     Warning: m_output_index is a deprecated keyword, please use output_component instead.
colvars:     # output_component = 0 [default]
colvars:     # doubleInputTensor = on
colvars:     Input dimension of model: 1
colvars:     Model's dtype: kFloat64.
colvars:     Evaluating model with zero tensor succeeded.
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 5
colvars:   # lowerBoundary = -180
colvars:   # upperBoundary = 180
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = on
colvars:   Enabling the extended Lagrangian term for colvar "nn_0".
colvars:   # extendedTemp = 300
colvars:   # extendedFluctuation = 5
colvars:   Computed extended system force constant: 0.0238463 [E]/U^2
colvars:   # extendedTimeConstant = 200
colvars:   Computed fictitious mass: 24.1613 [E]/(U/fs)^2   (U: colvar unit)
colvars:   # outputEnergy = off [default]
colvars:   # extendedLangevinDamping = 1 [default]
colvars:   # reflectingLowerBoundary = off [default]
colvars:   # reflectingUpperBoundary = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "abf" instance.
colvars:   # name = "abf1" [default]
colvars:   # colvars = { nn_0 }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 10 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   WARNING: ABF should not be run without a thermostat or at 0 Kelvin!
colvars:   # applyBias = on [default]
colvars:   # hideJacobian = off [default]
colvars:   Jacobian (geometric) forces will be included in reported free energy gradients.
colvars:   # fullSamples = 0
colvars:   # minSamples = 0 [default]
colvars:   # inputPrefix =  [default]
colvars:   # historyFreq = 0 [default]
colvars:   # shared = off [default]
colvars:   # updateBias = on [default]
colvars:   # maxForce =  [default]
colvars:   # CZARestimator = on [default]
colvars:   # integrate = on [default]
colvars:   # UIestimator = off [default]
colvars:   Finished ABF setup.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (4 scalable groups, 16 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - BAOA integrator:
colvars:   BouRabee2010 https://doi.org/10.1137/090758842
colvars: 
colvars: - Colvars module:
colvars: - Colvars-NAMD interface:
colvars: - Optimal rotation via flexible fitting:
colvars: - dihedral colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - linearCombination colvar component:
colvars:   Fiorin2024 https://doi.org/10.1021/acs.jpcb.4c05604
colvars: 
colvars: - ABF colvar bias implementation:
colvars:   Henin2010 https://doi.org/10.1021/ct9004432
colvars: 
colvars: - CZAR eABF estimator:
colvars: - eABF implementation:
colvars:   Lesage2017 https://doi.org/10.1021/acs.jpcb.6b10055
colvars: 
colvars: - NAMD engine:
colvars: - Scalable center-of-mass computation (NAMD):
colvars:   Phillips2020 https://doi.org/10.1063/5.0014475
colvars: 
colvars: updating target temperature (T = 0 K).
colvars: Updating NAMD interface:
colvars: updating atomic data (0 atoms).
colvars: updating group data (4 scalable groups, 16 atoms in total).
colvars: updating grid object data (0 grid objects in total).
colvars: updating target temperature (T = 0 K).
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "nn_0":0/0. 4 atoms: total mass = 54.028, total charge = -0.72.
colvars: Re-initialized atom group for variable "nn_0":0/1. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: Re-initialized atom group for variable "nn_0":0/2. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: Re-initialized atom group for variable "nn_0":0/3. 4 atoms: total mass = 54.028, total charge = -0.4.
colvars: The restart output state file will be "test.tmp.colvars.state".
colvars:   Initializing extended coordinate to colvar value.
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.tmp.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
